src/components/crossDevice/CrossDeviceSubmit/index.js

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'preact'
+
+import { trackComponent } from '../../../Tracker'
+import {preventDefaultOnClick} from '../../utils'
+import theme from '../../Theme/style.css'
+import style from './style.css'
+
+class CrossDeviceSubmit extends Component {
+  constructor (props) {
```

Comment:
empty constructor can be removed

Reply:
I've removed the constructors, good spot, I left them in while refactoring. Before those components had functions